From 0e6b51412cc8b166a8b4bd400637b2d9e84c0cfd Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Thu, 5 Jan 2017 14:20:55 +0800
Subject: [PATCH 1385/5242] MLK-13720-5: ARM: dts: imx7ulp-evk: support multi
 rpmsg instances

commit  a6de23d2da5f178ca26e70919a4d0d643e32f38f from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 'vdev-nums' to imply how many rpmsg instances used in kernel. If
it beyond the 5, please enlarge the 'MAX_VDEV_NUMS' in imx_rpmsg.c

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
(cherry picked from commit 0cbc0fdc4cce2c292aded56619d1001f7c936a6f)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index b2de9eb..622e6a4 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -401,6 +401,13 @@
 };
 
 &rpmsg{
+	/*
+	 * 64K for one rpmsg instance, default using 2 rpmsg instances:
+	 * --0x9FF00000~0x9FF01FFF: pingpong
+	 * --0x9FF10000~0x9FF11FFF: pmic
+	 */
+	vdev-nums = <2>;
+	reg = <0x9FF00000 0x20000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

